src/js/controllers/chatController.js

```diff
@@ -58,12 +60,30 @@ module.exports = ViewController.extend({
     },
 
     sendMessage: function(text) {
-        if (!!this.conversation) {
-            return this.conversation.get('messages').create({
+        var conversationDeferred = $.Deferred(),
+            messageDeferred = $.Deferred();
+
+        if (this.conversation.isNew()) {
+            this.conversation = this.collection.create(this.conversation, {
+                success: conversationDeferred.resolve,
+                error: conversationDeferred.reject
+            });
+
+            conversationDeferred.then(this._initFaye);
```

Comment:
While trying this, I think we don't want to throttle `updateUser` but debounce it with a maxWait. Currently, if you call updateUser and you are being throttle, you lose that information unless you reupdate the user outside of the throttle window.